docs/installation/linux/centos.md

```diff
@@ -29,101 +29,139 @@ your kernel must be 3.10 at minimum, which CentOS 7 runs.
 To check your current kernel version, open a terminal and use `uname -r` to
 display your kernel version:
 
-    $ uname -r
-    3.10.0-229.el7.x86_64
+```bash
+$ uname -r
+3.10.0-229.el7.x86_64
+```
 
 Finally, it is recommended that you fully update your system. Please keep in
 mind that your system should be fully patched to fix any potential kernel bugs.
 Any reported kernel bugs may have already been fixed on the latest kernel
 packages.
 
-## Install
+## Install Docker Engine
 
-There are two ways to install Docker Engine.  You can install using the `yum`
-package manager. Or you can use `curl` with the  `get.docker.com` site. This
-second method runs an installation script which also installs via the `yum`
-package manager.
+There are two ways to install Docker Engine.  You can [install using the `yum`
+package manager](#install-with-yum). Or you can use `curl` [with the  `get.docker.com`
+site](#install-with-the-script). This second method runs an installation script
+which also installs via the `yum` package manager.
 
 ### Install with yum
 
 1. Log into your machine as a user with `sudo` or `root` privileges.
 
-2. Make sure your existing yum packages are up-to-date.
+2. Make sure your existing packages are up-to-date.
 
-        $ sudo yum update
+    ```bash
+    $ sudo yum update
+    ```
 
 3. Add the yum repo.
 
-        $ sudo tee /etc/yum.repos.d/docker.repo <<-'EOF'
-        [dockerrepo]
-        name=Docker Repository
-        baseurl=https://yum.dockerproject.org/repo/main/centos/7/
-        enabled=1
-        gpgcheck=1
-        gpgkey=https://yum.dockerproject.org/gpg
-        EOF
+    ```bash
```

Comment:
Yeah you are right. I know in Asciidoc there are more elegant ways to do this. Not sure about Markdown.